bin/update_timestamp

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+
+require File.expand_path("../../config/environment", __FILE__)
+require "specialist_publisher"
+
+def update
+  $stderr.puts %Q{
+USAGE: #{File.basename(__FILE__)} (all|[format ...])(content_id) (date)
+
+1. available formats:
+
+  #{SpecialistPublisher.document_types.sort.join("\n  ")}
+
+2. content_id:
+	To find content_id, login to specialist-publisher	in production, go to content item page and copy content_id from url.
+
+3. date:
+	Format: yyyy/mm/dd or to specify publishing time: yyyy/mm/dd hh:mm
+	}
+
+  exit(1)
+end
+
+update unless ARGV.any?
+document_type = ARGV[0]
+content_id = ARGV[1]
+date = DateTime.parse(ARGV[2])
+
+raise ArgumentError, "please select document type among available formats provided" unless SpecialistPublisher.document_types.include?(document_type)
```

Comment:
Some of these `ArgumentError` lines're quite long. Maybe it'd be better to break them over multiple lines like